### PR TITLE
Removed need of shareDir & used pfstorage module

### PR DESCRIPTION
**Pull request by arnavn101**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -108,7 +108,9 @@ and then run
                 fnndsc/pman         \
                 --rawmode 1 --http  \
                 --port 5010         \
-                --listeners 12
+                --listeners 12      \
+                --storeBase=/tmp
```
Comment (husky-parul) [inline README.rst]: Add space 
`--storebase = /tmp`

File: README.rst [hunk 2]
```diff
@@ -195,4 +201,5 @@ Start ``pman`` with 12 listeners:
                 --rawmode 1                                         \\
                 --http                                              \\
                 --listeners 12                                      \\
+                --storeBase=/tmp                                    \\
```
Comment (husky-parul) [inline README.rst]: Space here

File: bin/pman [hunk 3]
```diff
@@ -108,6 +109,7 @@ def synopsis(ab_shortOnly = False):
                 --rawmode 1                                         \\
                 --http                                              \\
                 --listeners 12                                      \\
+                --storeBase=/tmp                                    \\
```
Comment (husky-parul) [inline bin/pman]: space here

File: bin/pman [hunk 4]
```diff
@@ -191,6 +198,7 @@ def synopsis(ab_shortOnly = False):
...
```
Comment (husky-parul) [inline bin/pman]: space here

File: pman/pman.py [hunk 5]
```diff
@@ -1827,8 +1843,9 @@ def t_run_process_container(self, *args, **kwargs):
                 str_managerApp              = d_manager['app']
 
                 d_env                       = d_manager['env']
-                if 'shareDir' in d_env.keys():
-                    str_shareDir            = d_env['shareDir']
+                if 'serviceType' in d_env.keys():
+                    global Gd_internalvar
```
Comment (husky-parul) [inline pman/pman.py]: why `global` ?

**Comment by arnavn101** [comment]
@danmcp  @husky-parul

File: pman/pman.py (hunk 5 above)
Comment (arnavn101) [reply pman/pman.py]: in pfioh.py, `global Gd_internalvar ` was always executed before referencing one of its elements

**Pull request closed without merging**